requirements/xpu.txt

```diff
@@ -1,17 +1,24 @@
 # Common dependencies
 -r common.txt
 
-ray >= 2.9
+ray>=2.9
 cmake>=3.26
 ninja
 packaging
 setuptools-scm>=8
 setuptools>=75.8.0
 wheel
 jinja2
+datasets # for benchmark scripts
 
-torch @ https://intel-optimized-pytorch.s3.cn-north-1.amazonaws.com.cn/ipex_dev/xpu/torch-2.5.0a0%2Bgite84e33f-cp310-cp310-linux_x86_64.whl
-intel-extension-for-pytorch @ https://intel-optimized-pytorch.s3.cn-north-1.amazonaws.com.cn/ipex_dev/xpu/intel_extension_for_pytorch-2.5.10%2Bgit9d489a8-cp310-cp310-linux_x86_64.whl
-oneccl_bind_pt @ https://intel-optimized-pytorch.s3.cn-north-1.amazonaws.com.cn/ipex_dev/xpu/oneccl_bind_pt-2.5.0%2Bxpu-cp310-cp310-linux_x86_64.whl
+torch==2.6.0+xpu
+torchaudio
+torchvision
+pytorch-triton-xpu
+--extra-index-url=https://download.pytorch.org/whl/xpu
 
-triton-xpu == 3.0.0b1
+# Please refer xpu doc, we need manually install intel-extension-for-pytorch 2.6.10+xpu due to there are some conflict dependencies with torch 2.6.0+xpu
+# FIXME: This will be fix in ipex 2.7. just leave this here for awareness.
+# intel-extension-for-pytorch==2.6.10+xpu
```

Comment:
Should a similar note be placed in the Dockerfile?

Reply:
added:)